### DISCUSSION TITLE
CSS problem with the homepage

### DISCUSSION THREAD

**Discussion by ocularvr** [mozilla/hubs#2978] [General Questions & Discussion] [answered]
The home page has a bug which is in the css:

.HomePage__logo-container__2V2SI {
    align-self: flex-start;
}

Should be

.HomePage__logo-container__2V2SI {
    align-self: center;
}

Can anyone help please? The result of this bug is that the company logo is not centered, as you can imagine this is the smallest thing but it causes the worst impression. Please help because I have reinstalled my stack four times, tried to find any css on the server, looked through the forum, read the documentation....

I will bite my lip and delete any emotional rants here.

**Comment by robinkwilson** [comment, marked as answer]
**Thumbs up this discussion if you want the logo to be default centered!** It's intentionally left aligned because the text isn't centered beneath it.

**To center it**
Add additional HTML/JS code in the admin panel: **Server Settings** menu > **Advanced** tab > HTML HomePage

Roughly off the top of my head add this HTML to the page you need to modify:

```
<style>
.HomePage__logo-container__2V2SI {
align-self: center;
}
</style>
```

**Reply by ocularvr** [reply]
Thank you Robin, that's exactly what I was after! The only problem now is to figure out where **Server Settings menu > Advanced tab > HTML HomePage** is, there are a few options. I have attached a screenshot. Thx
![hubs](https://user-images.githubusercontent.com/69241952/92586921-374d0780-f297-11ea-9d38-6457117fe351.jpg)

**Reply by robinkwilson** [reply]
"Extra HomePage Header HTML"

**Reply by giotti86** [reply]
I have copied and pasted but unfortunately it doesn't work

**Reply by robinkwilson** [reply]
@giotti86 Check which selector class your logo container is. It might be different than ocularvr's. 

`.HomePage__logo-container__2V2SI` --> **__2V2SI** is probably different for your build.

**Reply by ocularvr** [reply]
Just to add to Robins comment, my logo is set up like this:
`<div class="HomePage__logo-container__2V2SI HomePage__center-logo__2U_k9">`

**Reply by giotti86** [reply]
Thanks guys for the support;

in the end I succeeded by inserting:

@media (min-width: 992px)
.HomePage__logo-container__2V2SI {
    align-self:center!important;
}

**File path:**  Amazon S3/_yourprimarybouket_/hubs/assets/stylesheets
**File Name**: index-xxxxxx.css

**Reply by carogutt** [reply]
Thanks, @giotti86 works perfect!!!
Only thing, adding the curly brackets to contain what's on the media rule:

@media (min-width: 992px){
.HomePage__logo-container__2V2SI {
align-self:center!important;
}}
